corehq/apps/app_manager/views/app_summary.py

```diff
@@ -160,7 +160,7 @@ def _translate_name(names, language):
     if not names:
         return "[{}]".format(_("Unknown"))
     try:
-        return names[language]
+        return str(names[language])
```

Comment:
I think you should be using `six.text_type` here: https://pythonhosted.org/six/#six.text_type